pynance/dash_viz/plot_flow.py

```diff
@@ -0,0 +1,186 @@
+import base64
+import io
+
+import dash
+import flask
+
+from dash.dependencies import Input, Output, State
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+
+from pynance.textimporter import read_csv, SupportedCsvTypes
+
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']

```

Comment:
Why not include this stylesheet into the project? Would make it a bit more self-contained.

Reply:
it needed more than 5min for me to find out how, so I skipped it :) and I wasn't 100% sure, if I'm allowed to do this.
I thought we would need to develop a "corporate design" at some point anyway - and then can include this css then. What do you think?

Reply:
[](https://dash.plot.ly/external-resources)

Reply:
Yes, you are right. For the time being, I think we can leave it as is. And if we customise our design, we'll include our CSS. I'm resolving this discussion now.